canonical-web-and-design/jaas-dashboard
# Make WebCLI full width

## Done

Make WebCLI full width on Model Details page

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Enable WebCLI
- Verify it's full width across all breakpoints

## Details

Fixes #872
